fn_grpc_interface/fn_grpc_interface/util/grpc_helper.py

```diff
@@ -0,0 +1,165 @@
+        """
+        component that imports all the modules in the current workspace and returns the given imported modules objects.
+        :param interface_files: List of all buffer pb2 files absolute path to import in the current work space
+        :param base_dir:  interface pb2 files parent directory.
+        :param files_dir: interface pb2 files directory i.e same as package name.
+        :return: returns imported interface pb2 files module objects on the successful execution, else returns None object.
+        """
+        if not isinstance(interface_files, list):
+            raise ValueError("This Parameter "'"interface_files"'" should be list data type")
+        if base_dir is None:
+            raise ValueError("The param base_dir should contain parents parent directory of gRPC interface files")
+        if files_dir is None:
+            raise ValueError("The param files_dir should points to parent directory of gRPC interface files")
+
+        if interface_files:
+            abs_file_path_json = dict()
+            interface_module = []
+
+            # creating files abs path
+            for file_name in interface_files:
+                abs_file_path = os.path.join(base_dir, files_dir, file_name)
+                abs_file_path_json[file_name] = abs_file_path
+
+            # Adding Module path into sys.path variable
+            interface_dir_path = os.path.join(base_dir, files_dir)
+            if interface_dir_path not in sys.path:
+                sys.path.append(interface_dir_path)
+
+            # compatibility for python 2.7 and 3.6
+            self.log_object.info("Loading The Modules : {}".format(interface_files))
+
+            if six.PY3:
+                for file_name in interface_files:
+                    if file_name.endswith('.py'):
+                        module_name = file_name.split('.')[0]
+                        spec = importlib.util.spec_from_file_location(module_name, abs_file_path_json[file_name])
+                        if spec:
+                            module_obj = importlib.util.module_from_spec(spec)
+                            spec.loader.exec_module(module_obj)
+                            interface_module.append(module_obj)
+                    else:
+                        self.log_object.debug("No python modules found to import.")
+            elif six.PY2:
+                for file_name in interface_files:
+                    if file_name.endswith('.py'):
+                        module_name = file_name.split('.')[0]
+                        module_obj = imp.load_source(module_name, abs_file_path_json[file_name])
+                        interface_module.append(module_obj)
+            else:
+                raise ValueError("We do not support this version of python")
+
+            return interface_module
+        else:
+            self.log_object.debug("No Interface file found, please specify the interface file")
```

Comment:
Will the function work without an `interface file`? We are only logging a debug message here. Maybe we should raise a `ValueError`?

Reply:
yes